### ISSUE TITLE
Do not open a query or thread in a new tab if it is already open in a tab

### ISSUE THREAD

**Issue by mardukbp** [gauteh/astroid#149]
Sometimes it happens that I have a thread open, then I do other things and later on I want to view that thread again and open it in the thread index. Currently astroid opens a new tab with the thread even though it is already open in a tab. I would prefer it to switch to the corresponding tab instead of polluting the tab bar unnecessarily.

This also applies to queries.

**Comment by gauteh** [comment]
Yes. I think it should be configurable, it should not be too hard to
implement, you could try.

**Comment by hugoroy** [comment]
Honestly tabs are so easy to navigate and close, it does not seem that necessary :-)

**Comment by gauteh** [comment]
Hugo Roy writes on mars 2, 2017 0:29:
> Honestly tabs are so easy to navigate and close, it does not seem that necessary :-)

Maybe we could provide a Jump-to-tag search thing (with completion)?

**Comment by hugoroy** [comment]
↪ Gaute Hope / mars 2, 2017 8:31:
> Hugo Roy writes on mars 2, 2017 0:29:
>> Honestly tabs are so easy to navigate and close, it does not seem that necessary :-)
> 
> Maybe we could provide a Jump-to-tag search thing (with completion)?

I don't understand what that means :-)

**Comment by gauteh** [comment]
Hugo Roy writes on mars 2, 2017 9:25:
> ↪ Gaute Hope / mars 2, 2017 8:31:
>> Hugo Roy writes on mars 2, 2017 0:29:
>>> Honestly tabs are so easy to navigate and close, it does not seem that necessary :-)
>> 
>> Maybe we could provide a Jump-to-tag search thing (with completion)?
> 
> I don't understand what that means :-)
> 

So, say pressing M-b or something opens a search field like the regular
query or filter drop-down search bar. This can be used to search for
open tabs and will jump to the first one it matches.
